01/app.js

```diff
@@ -0,0 +1,6 @@
+const users = ['Jan Kowalski', 'Anna Nowak', 'Maciej Walczak', 'Michał Wójtowicz', 'Jessica Półrolniczak'];
+
+console.log(users[0]);
+console.log(users[2]);
+console.log(users[4]);
+console.log(users.length);
```

Comment:
Często można popełnić błąd na nazwie `.length` (poprzestawiać literki) - warto zawsze na to zwracać uwagę ;)